src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
I think that's a great idea (your second paragraph). Will push it up shortly! Just cleaning up. 
![image](https://cloud.githubusercontent.com/assets/16007382/26505702/cc4f86b6-4217-11e7-8e9d-0ae55d1c14be.png)